angularjs-autogrow.js

```diff
@@ -62,6 +62,30 @@
 
                 $scope.$watch($attrs.ngModel, $scope.autogrowFn);
 
+                // Extract css properties to spy on
+                var spyProps = $attrs.autogrow.split(',');
+                angular.forEach(spyProps, function(property) {
+                  // Set a watcher on each property
+                  $scope.$watch(
+                    function() {
+                      return $element.css(property);
+                    },
+                    styleChangedCallBack,
+                    true
```

Comment:
@frabnt looks great.
One question though: how necessary is the 3rd parameter of 'true' for deep watching?
Can you bring an example for it?